fuseki-docker/Dockerfile

```diff
@@ -0,0 +1,154 @@
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+## Apache Jena Fuseki server Dockerfile.
+
+## This Dockefile builds a reduced footprint container.
+
+ARG JAVA_VERSION=17
+
+ARG ALPINE_VERSION=3.17.1
+ARG JENA_VERSION=""
+ARG LOCK_UNLOCK_VERSION=""
+
+# Internal, passed between stages.
+ARG FUSEKI_DIR=/fuseki
+ARG FUSEKI_JAR=lock-unlock-fuseki-${JENA_VERSION}-${LOCK_UNLOCK_VERSION}.jar
```

Comment:
De app versie is nu "4.10.0-0.1.0", wat niet handig is ivm semver. Is ook niet heel logisch om versies van dependencies nog toe te voegen, simpelweg "0.1.0" volstaat IMHO

Moet ook in pom.xml aangepast worden, maar daar kon ik geen comment achterlaten

Reply:
Commit is coming up next